types/time/now.go

```diff
@@ -0,0 +1,13 @@
+//go:build !clock_skew
+// +build !clock_skew
```

Comment:
I wish these tags had a scope, like `ifdef ... endif` so that these two files, now and now_skew could be merged.

Reply:
Believe me, you don't want to go back to C 🤣